padertorch/data/segment.py

```diff
@@ -455,8 +555,17 @@ def moveaxis(tensor: torch.Tensor, source: int,
 
     num_samples = x.shape[axis]
     assert num_samples >= length, (num_samples, length)
-    if shift is None:
-        shift = length
+    if length_mode == 'constant':
+        end = 'cut'
+    elif length_mode in ['max', 'min']:
+        end = 'pad'
```

Comment:
I think we probably don't want to pad. In most cases, you want to ignore padded regions. Here, you don't know what is padded after calling this function.

Reply:
The max and min option were created for evaluation and in evaluation you should not use cut because it might change the signal.